src/main/java/roomescape/RoomescapeController.java

```diff
@@ -0,0 +1,64 @@
+package roomescape;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Controller
+public class RoomescapeController {
+    private List<Reservation> reservations = new ArrayList<>();
+
+    AtomicInteger atomic = new AtomicInteger(0);
+
+    @GetMapping("/")
+    public String home() {
+        return "home";
+    }
+
+    @GetMapping("/reservation")
+    public String reservation() {
+        return "reservation";
+    }
+
+    @GetMapping("/reservations")
+    @ResponseBody
+    public ResponseEntity<List<Reservation>> read() {
+        return ResponseEntity.ok(reservations);
+    }
+
+    @PostMapping("/reservations")
+    @ResponseBody
+    public ResponseEntity<Reservation> createReservation(@RequestBody Reservation reservation) {
+        if(reservation.getName().isEmpty() || reservation.getDate().isEmpty() || reservation.getDate().isEmpty()){
```

Comment:
reservation이 null이 온다면 어떤일이 벌어질까요?

Reply:
reservation이 null이 되어 이후 NullPointerException이 발생할 것으로 생각됩니다.
```suggestion
        if(reservation == null || reservation.getName().isEmpty() || reservation.getDate().isEmpty() || reservation.getDate().isEmpty()){
```
위와 같이 null에 대한 검사도 추가하면 에러처리가 가능할 것 같습니다.!

Reply:
브랜치 수정도 완료했습니다. 감사합니다!